src/nxt_conf_validation.c

```diff
@@ -1250,72 +1243,6 @@ nxt_conf_vldt_array_iterator(nxt_conf_validation_t *vldt,
     }
 }
 
-
```

Comment:
@VBart I removed the user/group validation at conf load-time because this will not be possible with isolation anymore.